src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -65,6 +67,12 @@
     public static final String ICE_CONNECTED_CHECK_SCRIPT =
         "return APP.conference.getConnectionState() === 'connected';";
 
+    /**
+     *  The javascript code which returns an array of remote participant IDs.
+     */
+    public static final String GET_REMOTE_PARTICIPANT_IDS =
+        "return APP.conference._room.getParticipants().map(p => p._id);";
```

Comment:
Do you think this is the best approach for the given desired changes? I'm generally not a fan of executing JS like this when it is avoidable, that way the end-to-end act more like a regular user. I think you can get the same info by scraping the dom, although then you rely on a stateful dom...so using JS seems okay too considering there's a lot of precedent for it already.

Reply:
Well, we are getting our own id using JS APP.conference ... so using the same way and getting remotes, I don't think is a problem.